hack/testing-olm/utils

```diff
@@ -13,6 +13,9 @@ gather_logging_resources() {
   local runtime=${3:-$(date +%s)}
   outdir=$outdir/$runtime
   mkdir $outdir ||:
-  oc adm must-gather --image=${IMAGE_CLUSTER_LOGGING_OPERATOR:-quay.io/openshift/origin-cluster-logging-operator:latest} --dest-dir=$outdir
+  oc -n ${LOGGING_NS} get deployments -o wide > $outdir/deployments.txt 2>&1 ||    :
+  oc -n ${LOGGING_NS} get ds -o wide > $outdir/daemonsets.txt 2>&1 ||    :
+  oc -n ${LOGGING_NS} get pods -o wide > $outdir/pods.txt 2>&1 || :
```

Comment:
The content of these files are in a different format already in must gather:
```
└── namespaces
   ├── openshift-logging
   │  ├── apps
   │  │  ├── daemonsets.yaml
   │  │  ├── deployments.yaml
   │  │  ├── replicasets.yaml
   │  │  └── statefulsets.yaml
```

Reply:
Pods on the other hand are structured as folder in must-gather:
```
│  ├── pods
   │  │  ├── cluster-logging-operator-74dd5994f-6ttgt
   │  │  │  ├── cluster-logging-operator
   │  │  │  │  └── cluster-logging-operator
   │  │  │  │     └── logs
   │  │  │  │        ├── current.log
   │  │  │  │        ├── previous.insecure.log
   │  │  │  │        └── previous.log
   │  │  │  └── cluster-logging-operator-74dd5994f-6ttgt.yaml
   │  │  ├── cluster-logging-operator-registry-6df49d7d4-mxxff
   │  │  │  ├── cluster-logging-operator-registry
   │  │  │  │  └── cluster-logging-operator-registry
   │  │  │  │     └── logs
   │  │  │  │        ├── current.log
   │  │  │  │        ├── previous.insecure.log
   │  │  │  │        └── previous.log
```

Reply:
having pods.txt in one place helps a lot in investigation.
i dont see openshift-logging namespace captured in the failed test case in this PR.

Reply:
I will add IMO the must gather format is not ideal for evaluating  a single sub system and requires you to search quite a bit though this is the trade-off for buying into it